index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Document</title>
+</head>
+
+<body>
+  <main id="main">
+    <h1> NGOZA & MARGARET</h1>
+    <p id="title">Microverse Trials</p>

```

Comment:
Please use html semantic tags. like header, nav or section tags depending on what specific type of group of elements. This is useful because it makes code easier for people to navigate when updating features or fixing bugs.
Giving your page a higher Search Engine Optimization (SEO) score etc.

Reply:
- [Optional] Kindly uses a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.